### PR TITLE
fix: design-system-old fixes

### PR DESCRIPTION
**Pull request by tanvibhakta**
## Description
Fixes
1. Gif player icon is now new colors
2. EditableText no longer contains old grays 

#### Type of change

- Bug fix (non-breaking change which fixes an issue)
>
- [ ] Manual
- [ ] Jest
- [ ] Cypress
>
>
#### Dev activity
- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] PR is being merged under a feature flag

#### QA activity:
- [ ] [Speedbreak features](https://github.com/appsmithorg/TestSmith/wiki/Guidelines-for-test-plans#speedbreakers-) have been covered
- [ ] Test plan covers all impacted features and [areas of interest](https://github.com/appsmithorg/TestSmith/wiki/Guidelines-for-test-plans#areas-of-interest-)
- [ ] Test plan has been peer reviewed by project stakeholders and other QA members
- [ ] Manually tested functionality on DP
- [ ] We had an implementation alignment call with stakeholders post QA Round 2
- [ ] Cypress test cases have been added and approved by SDET/manual QA
- [ ] Added `Test Plan Approved` label after Cypress tests were reviewed
- [ ] Added `Test Plan Approved` label after JUnit tests were reviewed

### CODE REVIEW

File: app/client/packages/design-system/widgets-old/src/themes/default/index.css [hunk 1]
```diff
@@ -38,59 +38,6 @@
   *
   */
 
-  /** API Pane */
```
Comment (albinAppsmith) [inline app/client/packages/design-system/widgets-old/src/themes/default/index.css:41]: Ensure this is not affecting any widgets.

**Review by albinAppsmith — APPROVED** [review]

**Comment by tanvibhakta** [comment]
@shadabbuchh please also sanity check the following widgets:

1. Currency Input (dropdown)
2. Dropdown widget
3. Input widget (currency code)
4. Input widget (ISD code)
5. Multi Select Tree Widget
6. Multi Select Widget V2 
7. Phone Input Widget (dropdown)
8. Rate Widget (tooltip)
9. Table Widget (scroll, search, header, )
10. LabelWithTooltip (tooltip)

**Review by jsartisan — DISMISSED** [review]

**Comment by shadabbuchh** [comment]
- Issue 1 - fixed

Screenshot 2023-06-14 at 12 25 56 PM

- Issue 2 - fixed

https://github.com/appsmithorg/appsmith/assets/39921438/92931ee7-1610-49f0-8d6d-13d8a9ade56c

However, the app name editableText (in the canvas) has a constant background (without interacting or hovering over it).
Screenshot 2023-06-14 at 12 51 57 PM


- Issue 3 - The error message needs to be a little more towards the left

Screenshot 2023-06-14 at 12 48 58 PM
Screenshot 2023-06-14 at 12 49 49 PM

**Comment by shadabbuchh** [comment]
> @shadabbuchh please also sanity check the following widgets:

@tanvibhakta this is blocked due to https://github.com/appsmithorg/appsmith/issues/24459

**Comment by tanvibhakta** [comment]
>    Issue 3 - The error message needs to be a little more towards the left

If we align it completely to the left it looks off as well. Let's keep this how it is for now.

> However, the app name editableText (in the canvas) has a constant background (without interacting or hovering over it).

This should be fixed, please take a look?

> @tanvibhakta this is blocked due to https://github.com/appsmithorg/appsmith/issues/24459

Does this issue need to block this PR from going in?

**Review by KelvinOm — DISMISSED** [review]

**Comment by shadabbuchh** [comment]
> If we align it completely to the left it looks off as well. Let's keep this how it is for now.

@tanvibhakta I disagree. If we shift it slightly towards the left it will look better. @vinay-appsmith , @vasanthappsmith any thoughts on this?

> However, the app name editableText (in the canvas) has a constant background (without interacting or hovering over it).

@tanvibhakta although we do not have the constant background, but now when we edit the name of the app the background of the editableText increases in width. Please have a look at the below video where I have done a comparison between the DP & Prod.


https://github.com/appsmithorg/appsmith/assets/39921438/80b70d18-e12c-47c3-98a3-ff84f793ce6a

**Comment by vinay-appsmith** [comment]
@tanvibhakta @shadabbuchh The error string needs to be aligned with the left-edge of the input field.

**Review by albinAppsmith — APPROVED** [review]

**Pull request merged**